mslib/mscolab/models.py

```diff
@@ -191,6 +191,7 @@ def __init__(self, op_id, u_id, text, message_type=MessageType.TEXT, reply_id=No
         self.text = text
         self.message_type = message_type
         self.reply_id = reply_id
+        self.created_at = datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
we need to check if this is needed, or why it makes a difference

![develop](https://github.com/Open-MSS/MSS/assets/278728/37b7404b-db80-4aa0-92d2-a92082db68bb)